src/main/scala/org/opennetworkinsight/LDAWrapper.scala

```diff
@@ -35,13 +34,13 @@ object LDAWrapper {
       words.zipWithIndex.toMap
     }
 
-    val distinctDocument = documentWordData.map(row => row(0)).distinct
-    distinctDocument.cache()
+    val distinctDocument = documentWordData.map(row => row(0)).distinct.collect
+    //distinctDocument.cache()
```

Comment:
set of distinct documents should fit in memory easy, so I opted for simpler solution